lib/init.coffee

```diff
@@ -2,10 +2,31 @@ module.exports =
   config:
     javaExecutablePath:
       type: 'string'
-      default: ''
+      default: 'java'
     clojureExecutablePath:
       type: 'string'
-      default: ''
+      default: 'clojure-x.x.x.jar'
 
-  activate: ->
-    console.log 'activate linter-clojure'
+  provideLinter: ->
+    helpers = require('atom-linter')
+    regex = 'RuntimeException:(?<message>.*), compiling:(.*):(?<line>\\d+):(?<col>\\d+)'
+    provider =
+      grammarScopes: ['source.clojure', 'source.clojurescript']
+      scope: 'file'
+      lintOnFly: true
+      lint: (textEditor) =>
+        filePath = textEditor.getPath()
+        command = atom.config.get('linter-clojure.javaExecutablePath') or 'java'
+        parameters = [
+          '-jar',
+          atom.config.get('linter-clojure.clojureExecutablePath'),
+          '-i',
+          filePath
+        ]
+
+        return helpers.exec(command, parameters, {stream: 'stderr'}).then (output) ->
+          errors = for message in helpers.parse(output, regex, {filePath: filePath})
```

Comment:
In which function do you see that you can set a default `type`? 

https://github.com/AtomLinter/atom-linter/blob/master/lib/helpers.coffee#L78